introduction.md

```diff
@@ -0,0 +1 @@
+Hello everyone. My name is Phil. And I am new to this dojo. Thank you for having me
```

Comment:
So far I have only explored the Git path, so by default it is my favorite. I wish there were a Java path;p